### DISCUSSION TITLE
How Can I disable the pop-up window (Enter visit purpose)?

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#15295] [How to] [answered]
Hi guys,

Every time that I log in to the YetiForce CRM, the system shows a pop-up window (Enter visit purpose). How could I disable this?

I have tried to find the config in the Configuration setup and I am unable to find it.

Thank you

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/14497
https://github.com/YetiForceCompany/YetiForceCRM/search?q=Visit+Purpose&type=issues
https://github.com/YetiForceCompany/YetiForceCRM/blob/b73408496b9bfd8b2c7a09a04d6f6bbb9e4867b7/config/Security.php#L202-L209

**Reply by rskrzypczak** [reply]
![image](https://user-images.githubusercontent.com/10221999/136831411-62b56531-3019-47cc-89cd-a9772359ccdc.png)
